internal/cmd/base/delete.go

```diff
@@ -34,7 +34,7 @@ func (dc *DeleteCmd) CobraCommand(s state.State) *cobra.Command {
 	cmd := &cobra.Command{
 		Use:                   fmt.Sprintf("delete %s<%s>", opts, strings.ToLower(dc.ResourceNameSingular)),
 		Short:                 dc.ShortDescription,
-		Args:                  cobra.ExactArgs(1),
+		Args:                  util.ValidateExact,
```

Comment:
The Base Command already sets a default for every command that does not define `Args`: https://github.com/hetznercloud/cli/blob/b9265f42ad57d8f35e28c30ef7e3e45cb2341753/internal/cmd/base/cmd.go#L21-L23

You could update this default to `util.ValidateExact` and remove `Args: util.ValidateExact` from every command that uses `base.Cmd` directly or through `base.ListCmd`, `base.CreateCmd` (...).

A potential follow up could be to make sure that every command uses `base.Cmd`, at a quick glance it was mostly the no-op parent commands that made no use of it so far. perhaps it makes sense to introduce a no-op `base.ParentCmd` that only hosts children. That way we could also remove some other defaults set in `base.Cmd` from all actual commands: `TraverseChildren` and `DisableFlagsInUseLine`.